common/script/index.coffee

```diff
@@ -1382,14 +1391,16 @@ api.wrap = (user, main=true) ->
           user.stats.lvl++
           tnl = api.tnl(user.stats.lvl)
 
+          user.stats.hp = 50
+
+          return if user.stats.lvl > api.maxLevel
```

Comment:
Good catch. I don't have a chance to compile it now, but I wonder if this would work as well: `continue if user.stats.lvl > api.maxLevel`. That way, we wouldn't have to wrap a whole section of code in an `if...` block, but we get the same effect.